src/lib/network_pool/snark_pool.ml

```diff
@@ -367,14 +367,19 @@ struct
           let account_opt =
             let open Mina_base in
             let open Option.Let_syntax in
-            let%bind ledger = best_tip_ledger t in
+            let btl = best_tip_ledger t in
+            [%log' debug t.logger] "Best tip ledger: should be TRUE: $ledger"
+              ~metadata:[ ("ledger", `Bool (Option.is_some btl)) ] ;
+            let%bind ledger = btl in
             if Deferred.is_determined (Base_ledger.detached_signal ledger) then
               None
             else
               let%bind loc =
+                (* how can you be sure the token id will be MINA ? This seems like a pre-berk assumption *)
```

Comment:
token id for snark fees? Snark pool diffs don't encode any token ID. So whatever is the fee it is taken to be in MINA

Reply:
removing